client/signup/config/flows-pure.js

```diff
@@ -281,7 +281,7 @@ export function generateFlows( {
 	};
 
 	flows.import = {
-		steps: [ 'from-url', 'user', 'domains' ],
+		steps: [ 'from-url', 'user', 'domains', 'plans-without-ecommerce' ],
```

Comment:
How about `plans-import`? Gives flexibility, since what plans we show for the flow could evolve over time.